utilities/pipelines/publish/helper/Get-ModulesToPublish.ps1

```diff
@@ -188,14 +211,17 @@ function Get-ModulesToPublish {
     [CmdletBinding()]
     param (
         [Parameter(Mandatory)]
-        [string] $ModuleFolderPath
+        [string] $ModuleFolderPath,
+
+        [Parameter(Mandatory = $false)]
+        [switch] $SkipNotVersionedModules
     )
 
     $versionFile = (Get-Content (Join-Path $ModuleFolderPath 'version.json') -Raw) | ConvertFrom-Json
     $PathsToInclude = $versionFile.PathFilters
 
     # Check as per a `diff` with head^-1 if there was a change in any file that would justify a publish
-    $TemplateFilesToPublish = Get-TemplateFileToPublish -ModuleFolderPath $ModuleFolderPath -PathsToInclude $PathsToInclude
+    $TemplateFilesToPublish = Get-TemplateFileToPublish -ModuleFolderPath $ModuleFolderPath -PathsToInclude $PathsToInclude -SkipNotVersionedModules
 
     # Filter out any children (as they're currently not considered for publishing)
```

Comment:
While we're touching the file anyways we can maybe remove this clearly outdated code